evap/templates/base.html

```diff
@@ -160,6 +160,37 @@
         {% endblock %}
     </div>
 </div>
+
+{% if user.is_authenticated and not user.is_staff %}
```

Comment:
please show for staff as well. sorry that i didn't specify before. we don't have any features that are disabled for staff yet and we shouldn't start now. besides, staff users are also students and not all of them actively organize the evaluations, so they should also be able to use the feedback form.